source/content/apps/velja.md

```diff
@@ -373,19 +373,13 @@ It means [“to choose”](https://en.wiktionary.org/wiki/velja) in [Old Norse](
 Velja benefits:
 
 - Free
-- It can [open links to certain services](https://twitter.com/sindresorhus/status/1519020970027401216) in their desktop app (Zoom, Microsoft Teams, Figma, etc.)
```

Comment:
This is about apps that don't simply accept a URL, but needs some kind of transformation. If Choosy supports this, can you link to a list of apps it supports?

Reply:
Choosy can only send links to apps that support opening HTTP URLs. I definitely think you need to make this item clearer, because it sounds like a really cool feature 🙂